### ISSUE TITLE
Word format report cannot be generated when <br > tag is used

### ISSUE THREAD

**Issue by underfl0w** [GhostManager/Ghostwriter#82]
## Expected Behavior
Generate a .docx file without any issue, either ignoring the `<br>` tag or by adding a linebreak. 

## Current Behavior
An error shows in the web interface stating: 
`Failed to generate the Word report for an unknown reason: list index out of range`
The docker logs show the error 
`Unknown nested tag - br`

## Steps to Reproduce

1. Create a new report 
2. Add a finding and add the `<br>` tag to one of the fields 
3. Generate a report 

## Context (Environment)
I am using the latest code from Github and running everything in the production docker image. 

## Detailed Description
The issue started to appear after copy and pasting some output from my terminal into one of the fields. The WYSIWYG text editor automatically added the` <br />` tags and caused the issue to appear.

**Comment by chrismaddalena** [comment]
Thanks for reporting this, @underfl0w. The `Unknown nested tag - br` is just some logging text specifically for situations like this. It mens Ghostwriter saw the tag and ignored it, so the tag should not have caused any problems. The error may be related to the placement of the tag, like inside of a list or something like that.

It's also strange that the WYSIWYG editor inserted that tag. I just tried editing source and manually add `<br />` to a line and TinyMCE converts that to `<p>&nbsp;</p>` without asking. I experimented with it and could add `<br />` only if I manually inserted it in between some words inside of another tag, like `<p></p>`. I didn't have any issues generating the report after that. Ghostwriter identified the `<br />` tag and ignored it.

My thought is the problem is related to where the `<br />` tag is appearing. You mentioned it appeared after copy/pasting from your terminal. Where did you paste the text – i.e., did you paste it into a list item or another element?

I'll look into it, but know some details about the placement of the tag will help us zero in on the problem.
